IPAdapterPlus.py

```diff
@@ -400,6 +442,7 @@ def INPUT_TYPES(s):
                 "start_at": ("FLOAT", { "default": 0.0, "min": 0.0, "max": 1.0, "step": 0.001 }),
                 "end_at": ("FLOAT", { "default": 1.0, "min": 0.0, "max": 1.0, "step": 0.001 }),
                 "unfold_batch": ("BOOLEAN", { "default": False }),
+                "face_id": ("BOOLEAN", { "default": False }),
```

Comment:
Perhaps you can figure out how to detect the model at runtime. I didn't bother to.